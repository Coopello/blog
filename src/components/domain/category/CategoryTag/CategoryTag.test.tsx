import renderer from "react-test-renderer";

import { CategoryTag } from ".";

describe("domain/category/ArticleTag", () => {
  it("Snap Shot", () => {
    const component = renderer.create(
      <CategoryTag type="front-end" text="front-end" />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
