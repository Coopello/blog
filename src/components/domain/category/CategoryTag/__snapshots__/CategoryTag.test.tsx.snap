// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`domain/category/ArticleTag Snap Shot 1`] = `
<span
  className="rounded-sm text-sm"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  style={
    {
      "backgroundColor": "white",
      "color": "#E10718",
      "paddingLeft": 8,
      "paddingRight": 8,
    }
  }
>
  #
  front-end
</span>
`;
